app/assets/scripts/views/location.js

```diff
@@ -240,30 +240,39 @@ var Location = React.createClass({
   },
 
   renderSourceInfo: function () {
+    const {data} = this.props.measurements;
+
     let sources = this.props.loc.data.sourceNames
       .map(o => _.find(this.props.sources, {name: o}))
       .filter(o => o);
 
+    if (data.attribution) {
+      // filtering attribution[0] b/c it kept showing up with same name and url as sources[0]
+      let added = data.attribution.filter((src, index) => index !== 0);
+      sources = [...sources, ...added];
+    }
+
     return (
       <section className='fold fold--filled' id='location-source'>
         <div className='inner'>
-          <header className='fold__header'>
-            <h1 className='fold__title'>Source information</h1>
+          <header className=''>
+            <h5 className='fold__title'>Sources:</h5>
           </header>
           <div className='fold__body'>
+            <div className='col-main'>
             <ul className='sources__list'>
-              {sources.map(source => <li key={source.name}>
-                <div className='col-main'>
-                  <p>Source: <a href={source.sourceURL} title='View source information'>{source.name}</a></p>
-                </div>
-                <div className='col-sec'>
-                  {source.description
-                    ? <p>{source.description}</p>
-                    : null}
-                  For more information contact <a href={`mailto:${source.contacts[0]}`}>{source.contacts[0]}</a>.
-                </div>
-              </li>)}
+              {sources.map(source =>
+                <li key={source.name}>
+                    <p><a href={source.sourceURL || source.url} title='View source information'>{source.name}</a></p>
```

Comment:
Not all attribution objects will have a URL. When this is the case, could you not use the <a> element so it doesn't show up as a link?